pkg/services/object/search/remote.go

```diff
@@ -19,6 +19,8 @@ type remoteStream struct {
 	addr *network.Address
 
 	clientCache *cache.ClientCache
+
+	clientOpts []client.Option
```

Comment:
This field is not set in the code. See `pkg/services/object/search/streamer.go:163`

Reply:
Fixed, thx.